quickjs.c

```diff
@@ -37208,6 +37208,13 @@ static JSValue js_object_isSealed(JSContext *ctx, JSValue this_val,
     return JS_EXCEPTION;
 }
 
+JSValue JS_ObjectSeal(JSContext *ctx, JSValueConst obj, int freeze)
+{
+    JSValueConst argv[] = {obj};
+    JS_FreeValue(ctx, js_object_seal(ctx, JS_UNDEFINED, 1, argv, freeze));
+    return obj;
+}
```

Comment:
We've phased out JSValueConst (it's just an alias for JSValue now) so this can be changed to:
```suggestion
JSValue JS_ObjectSeal(JSContext *ctx, JSValue obj, int freeze)
{
    JS_FreeValue(ctx, js_object_seal(ctx, JS_UNDEFINED, 1, &obj, freeze));
    return obj;
}
```